tests/test_orders.py

```diff
@@ -1,22 +1,21 @@
 import requests
+import requests_mock
+
 from woocommerce import API
 
 from mock_data.mock_order import order
+from app import woo_get
+
+BASE_URL = "https://www.anything.com"
 
-def test_list_orders(mocker):
-    url = "https://www.anything.com"
-    ck = "test_key"
-    cs = "test_secret"
-    wcapi = API(
-        url=f"{url}",
-        consumer_key=f"{ck}",
-        consumer_secret=f"{cs}",
-        version="wc/v3"
-    )
-    mocker.patch('requests.get')
```

Comment:
somehow this did not work, we hit the anything.com site, the real one, so response was 404 because it has no orders page

we debugged `wcapi.get` and it seems it uses `requests.request("GET")`, not sure if it matters but changing the argument here did not work either, so we switched to `requests-mock`, then it worked